roles/install/tasks/apt/install_one.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Latest component version
+  set_fact:
+    pkg_name: '{{ item.name }}'
+    pkg_state: latest
+  when: item.version == 'latest'
+
+- name: Specific component version ({{ item.version }})
+  set_fact:
+    pkg_name: '{{ item.name }}={{ item.version }}-*'
+    pkg_state: present
+  when: item.version != 'latest'
+
+- name: Install component ({{ pkg_name }})
+  apt:
```

Comment:
I am pretty sure this command will also start the service on Ubuntu 14.04. Maybe we should set the `RUNLEVEL` env variable to `1` in order to keep the services down?